core/commands/config.go

```diff
@@ -332,7 +332,7 @@ can't be undone.
 		}
 		defer file.Close()
 
-		return replaceConfig(r, file)
+		return replaceConfig(r, io.LimitReader(file, 32*1024*1024)) // 32MiB
```

Comment:
I see we use this 32MiB limit in more than one place, so you mind refactoring this and https://github.com/ipfs/kubo/pull/9381 to use the same constant?